django_qbe/exports.py

```diff
@@ -43,11 +43,16 @@ def __init__(self, dialect=csv.excel_tab, encoding="utf-8", **kwds):
         self.queue = BytesIO() if six.PY2 else StringIO()
         self.writer = csv.writer(self.queue, dialect=dialect, **kwds)
 
-    def writerow(self, row):
+    def _encode(self, item):
         if six.PY2:
-            self.writer.writerow([s.encode('utf-8') for s in row])
+            encoded = unicode(item).encode('utf-8')
         else:
-            self.writer.writerow([s for s in row])
+            encoded = str(item)
+
+        return encoded
```

Comment:
Cool, this works.

Did you consider the 3 lines shorter version with 2 returns and no else?

Reply:
I did, but I find this a bit easier to read.

Reply:
I generally prefer shorter, less typos, less bugs, but personal preference allowed!